client/src/components/Review/DetailsPanel.js

```diff
@@ -32,42 +86,64 @@ class DetailsPanel extends React.Component {
       });
   }
 
+  // RENDER
   render() {
     return (
       <StyleModal>
         <Header> Details </Header>
+      {/* // -- // */}
         <Content>
           {" "}
           <h2>Name:</h2>
-          <p>{this.props.details[0]}</p>
+            <p>{this.props.details[0]}</p>
           <h2>Phone:</h2>
-          <p>{this.props.details[1]}</p>
-          <h2>Hours:</h2>
-          {this.props.hours.map(day => {
-            return <p>- {day} -</p>;
-          })}
-          {this.state.location_id.map(location => {
-            return (
-              <ul key={location.id}>
-                <li>
-                  {" "}
-                  <p>UserId: {location.user_id},</p>{" "}
-                </li>
-                <li>
-                  {" "}
-                  <p>Rating: {location.rating},</p>{" "}
-                </li>
-                <li>
-                  {" "}
-                  <p>Comments: {location.comment}</p>{" "}
-                </li>
-              </ul>
-            );
-          })}
+            <p>{this.props.details[1]}</p>
+          <h2 className='hours'>Hours:</h2>
+          <ul>
+            {this.props.hours.map(day => {
+              return <li><div>{day}</div></li>;
```

Comment:
also needs a key, would also restructure this to make sense, I don't see how this.props.hours is expected to return a day